src/app/edit-maprule/disabled-feature/disabled-feature.html

```diff
@@ -0,0 +1,55 @@
+<div #disabledFeature class="disabled-feature-container">
+  <div class="disabled-feature-card">
+    <div class="disabled-feature-header disabled-feature-card-panel-header-hidden">
+      <div>
+        <button class="preset-button" id="show-preset-button" (click)="showDisabledFeaturePanel()">
+          <i class="fa fa-plus-square-o"></i>
+        </button>
+      </div>
+    </div>
+    <div class="preset-card-panel preset-card-panel-hidden">
+      <div [formGroup]="editMapRule.form" class="content-group disabled-feature-panel">
+          <div class="table-container disabled-feature-table-container"
```

Comment:
This new sort of table is used also in the preset template.
The bootstrap table, while the O.G., just made it hard to show all the selected values w/the new comboboxes...
I put the link in a comment, but this came from some codepen I found!